pkgs/tools/misc/calamares/default.nix

```diff
@@ -1,24 +1,26 @@
 { lib, fetchurl, boost, cmake, extra-cmake-modules, kparts, kpmcore
-, kservice, libatasmart, libxcb, libyamlcpp, parted, polkit-qt, python, qtbase
-, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
-, ckbcomp, xkeyboard_config, mkDerivation
+, kservice, libatasmart, libxcb, libyamlcpp, libpwquality, parted, polkit-qt, python
+, qtbase, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
+, ckbcomp, xkeyboard_config, mkDerivation, nixos-extensions ? false
 }:
 
 mkDerivation rec {
   pname = "calamares";
-  version = "3.2.44.3";
+  version = "3.2.49.1";
 
   # release including submodule
   src = fetchurl {
     url = "https://github.com/${pname}/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "sha256-p3ctULrzXPt9dNs8Ckb7cqdOBpp4qOmEwu0dEVq8lEw=";
+    sha256 = "sha256-hr3OuQ64HUJZa3gMzZk4UALiuNRdj66DYVbTe1Ei0CA";
   };
```

Comment:
This should use fetchFromGitHub.

Reply:
Stuff under `releases/download/` are dist tarballs uploaded by upstream, not the tarballs automatically generated by GitHub (those are under `archive/`). So `fetchurl` is in fact correct here.